ncm-filesystems/src/main/pan/components/filesystems/schema.pan

```diff
@@ -8,10 +8,14 @@ include 'quattor/schema';
 include 'quattor/blockdevices';
 include 'quattor/filesystems';
 
+@documentation{
+when manage_blockdevs is false, filesystems does same as fstab
```

Comment:
well, not having the code in ncm-fstab was the reason why I closed #600 : 'Agreement from the workshop: refactoring the duplicated code into ncm-lib-blockdevices or somewhere else is a better and safer way. RAL and Morgan Stanley still want to keep the dangerous code isolated and if possible not even installed.'
running both fstab and filesystems will first do filesystems, then run fstab (so fstab code will run twice, as if you run the component twice)